pages/index.js

```diff
@@ -33,13 +43,18 @@ export default function HomePage() {
       <StyledHeadline>List of Activities</StyledHeadline>
       <StyledUl>
         {activities.map((activity) => (
+          <>
```

Comment:
you don't need these fragments.

Reply:
We tried to remove them, but got errors und we decided that we need them 😊